packages/cli/src/commands/base-command.ts

```diff
@@ -294,6 +294,8 @@ export abstract class BaseCommand extends Command {
 			this.logger.info(`Received ${signal}. Shutting down...`);
 			this.shutdownService.shutdown();
 
+			await this.errorReporter.shutdown();
```

Comment:
Should this be done after shutdownService has shutdown? I think we might possibly lose errors happening during shutdown